zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -199,6 +199,7 @@ pub fn create_spend_to_address<DbT, ParamsT>(
     memo: Option<MemoBytes>,
     ovk_policy: OvkPolicy,
     min_confirmations: NonZeroU32,
+    change_memo: Option<MemoBytes>,
```

Comment:
The doc comment needs to be updated in the Parameters section and the example (lines 124 and 181).